No idea why, but manually importing the gst module works, but from the
configure script it fails with:

Traceback (most recent call last):
  File "depcheck.py", line 86, in <module>
    check_dependencies()
  File "depcheck.py", line 71, in check_dependencies
    import gst
  File "/usr/lib/python2.6/site-packages/gst-0.10/gst/__init__.py", line 170, in <module>
    from _gst import *
RuntimeError: can't initialize module gst: Error re-scanning registry , child terminated by signal

so just don't check for it for now.

diff --git a/depcheck.py b/depcheck.py
index 3325b5c..2a78b54 100644
--- a/depcheck.py
+++ b/depcheck.py
@@ -67,17 +67,6 @@ def check_dependencies():
 		except ImportError:
 			print "Error: python-musicbrainz2 was not found."
 			raise SystemExit
-		try:
-			import gst
-			if gst.version() < (0, 10, 3, 0):
-				raise AssertionError
-			print "Checking for Gstreamer (Python)... yes"
-		except ImportError:
-			print "Error: gst-python was not found"
-			raise SystemExit
-		except AssertionError:
-			print "Error: Version 0.10.3 or better of Gstreamer needed."
-			raise SystemExit
 	except SystemExit:
 		from sys import exit
 		exit(1)
